pyroSAR/gamma/parser_demo.py

```diff
@@ -2906,7 +2906,7 @@ def par_KC_PALSAR_slr(facter_m, CEOS_leader, SLC_par, pol, pls_mode, KC_data, pw
         a file to write the Gamma commands to in shell format
     """
     process(
```

Comment:
please remove this. `parser_demo.py` is not needed and should not be changed by hand. The automatically parsed Gamma functions can be found under `$HOME/.pyrosar/gammaparse`.